atomics/T1218/T1218.yaml

```diff
@@ -223,3 +223,30 @@ atomic_tests:
   executor:
     command: 'Invoke-ATHRemoteFXvGPUDisablementCommand -ModuleName #{module_name} -ModulePath #{module_path}'
     name: powershell
+- name: DiskShadow Command Execution
+  description: |
+    Emulates attack with a DiskShadow.exe being used to execute arbitrary commands
+  supported_platforms:
+    - windows
+  input_arguments:
+    txt_payload:
+      description: txt to execute
+      type: Path
+      default: PathToAtomicsFolder\T1218\src\T1218.txt
+    dspath:
+      description: Default location of DiskShadow.exe
+      type: Path
+      default: C:\Windows\System32\diskshadow.exe
+  dependency_executor_name: powershell 
+  dependencies: 
+    - description: txt file must exist on disk at specified location (#{txt_payload})
+      prereq_command: |
+        if (Test-Path #{txt_payload}) {exit 0} else {exit 1}
+      get_prereq_command: |
+        New-Item -Type Directory (split-path #{txt_payload}) -ErrorAction ignore | Out-Null
+        Invoke-WebRequest "https://github.com/redcanaryco/atomic-red-team/raw/master/atomics/T1218/src/T1218.txt" -OutFile "#{txt_payload}"
```

Comment:
Could you add a dependency that will check that diskshadow.exe exists at `dspath`? You don't necessarily need to automate the get_prereq part, you could just print a msg that no automated steps to install are provided. Also, can you make a note somewhere that diskshadow is included by default in Windows Server?

Reply:
Thanks for the feedback, I've added a second prereq test to verify the existence of DiskShadow.exe at its default path, and added wording to the description to clarify that this is a Microsoft shipped-by-default binary in Windows Server.